### PR TITLE
201507 MMH updates

### PR DESCRIPTION
**Pull request by julianorrish**

### CODE REVIEW

File: maternal-mental-health/test-4.md [hunk 1]
```diff
@@ -29,19 +29,19 @@ Please choose the ONE most correct answer to each question:
...
 	-	Having an increased appetite
 	-	Positive reaction from important people
 	+	Relationship problems with partner
 	-	Improved sex life
-7.	‘De-personalised’ means:
+7.	An example of ‘de-personalisation’ is:
 	-	Denying personal attention
-	+	Referring to someone by a number
-	-	Defying the person in charge
+	+	Referring to someone as "patient" or by a number
```
Comment (arthurattwell) [inline maternal-mental-health/test-4.md]: I think our style is single quotes?

File: maternal-mental-health/test-6.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ Please choose the ONE most correct answer to each question:
 	-	Internal perforation – violent
 	-	Indigo, purple, violet – bruise categories
 	-	Independent personal victim
-	+	intimate partner violence
+	+	Intimate partner violence
```
Comment (arthurattwell) [inline maternal-mental-health/test-6.md]: Need to hyphenate 'Intimate-partner'.

**Comment by arthurattwell** [comment]
Merged. Can I delete the branch?

**Comment by julianorrish** [comment]
Yes :)

File: maternal-mental-health/test-4.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ Please choose the ONE most correct answer to each question:
 	-	Improved sex life
 7.	An example of ‘de-personalisation’ is:
 	-	Denying personal attention
-	+	Referring to someone as "patient" or by a number
+	+	Referring to someone as 'patient' or by a number
```
Comment (arthurattwell) [inline maternal-mental-health/test-4.md:43]: This is a test comment on a pull request.

**Pull request merged**